cve_bin_tool/extractor.py

```diff
@@ -90,7 +90,7 @@ def extract_file_rpm(cls, filename, extraction_path):
             if not inpath("7z"):
                 raise Exception("7z is required to extract rpm files")
             else:
-                subprocess.call("7z x " + filename + " -o" + extraction_path)
+                subprocess.call(f"7z x {filename} -o {extraction_path}")
```

Comment:
Not your fault, but we should probably switch this to use subprocess with the arguments in an array, since that's the preferred method now.  I'll open another issue to look at this and see if it can be converted.